coffea/ml_tools/triton_wrapper.py

```diff
@@ -0,0 +1,495 @@
+# For python niceties
+import abc
+import warnings
+from typing import Dict, List, Optional, Union
+
+import awkward
+import dask_awkward
+
+# For data manipulation
+import numpy
+
+# For triton specific handling
+try:
+    import tritonclient.grpc
+    import tritonclient.http
+    import tritonclient.utils
+except ImportError as err:
```

Comment:
What would be the best practice here? 
Just a `pass` statement?

Reply:
Add a message to the exception that says what needs to be done in order to use this module and reraise!